### ISSUE TITLE
Fix duplicate Wyvern pet name values

### ISSUE THREAD

**Issue by claywar** [DerpyProjectGroup/topaz#251]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

CI Result:
```
    scripts/globals/pets.lua:99:5: (W314) value assigned to field AZURE is overwritten on line 575 before use
    scripts/globals/pets.lua:100:5: (W314) value assigned to field CERULEAN is overwritten on line 1078 before use
    scripts/globals/pets.lua:104:5: (W314) value assigned to field EMBER is overwritten on line 612 before use
    scripts/globals/pets.lua:106:5: (W314) value assigned to field MAX is overwritten on line 439 before use
    scripts/globals/pets.lua:108:5: (W314) value assigned to field DUKE is overwritten on line 472 before use
    scripts/globals/pets.lua:112:5: (W314) value assigned to field LADY is overwritten on line 507 before use
    scripts/globals/pets.lua:114:5: (W314) value assigned to field RAIDEN is overwritten on line 869 before use
    scripts/globals/pets.lua:117:5: (W314) value assigned to field PFEIL is overwritten on line 686 before use
    scripts/globals/pets.lua:149:5: (W314) value assigned to field KOMET is overwritten on line 656 before use
    scripts/globals/pets.lua:154:5: (W314) value assigned to field BARON is overwritten on line 576 before use
    scripts/globals/pets.lua:155:5: (W314) value assigned to field GINGER is overwritten on line 799 before use
    scripts/globals/pets.lua:156:5: (W314) value assigned to field MUFFIN is overwritten on line 282 before use
    scripts/globals/pets.lua:170:5: (W314) value assigned to field ETOILE is overwritten on line 779 before use
    scripts/globals/pets.lua:171:5: (W314) value assigned to field TONNERRE is overwritten on line 1149 before use
    scripts/globals/pets.lua:172:5: (W314) value assigned to field NUAGE is overwritten on line 674 before use
    scripts/globals/pets.lua:175:5: (W314) value assigned to field ORAGE is overwritten on line 681 before use
    scripts/globals/pets.lua:176:5: (W314) value assigned to field LUNE is overwritten on line 516 before use
    scripts/globals/pets.lua:179:5: (W314) value assigned to field SOLEIL is overwritten on line 898 before use
    scripts/globals/pets.lua:185:5: (W314) value assigned to field CELESTE is overwritten on line 940 before use
    scripts/globals/pets.lua:189:5: (W314) value assigned to field ORAGEUSE is overwritten on line 1126 before use
    scripts/globals/pets.lua:254:5: (W314) value assigned to field SAKURA is overwritten on line 882 before use
    scripts/globals/pets.lua:281:5: (W314) value assigned to field GUSTAV is overwritten on line 808 before use
    scripts/globals/pets.lua:293:5: (W314) value assigned to field CLOCHETTE is overwritten on line 1167 before use
    scripts/globals/pets.lua:295:5: (W314) value assigned to field KAISERIN is overwritten on line 1111 before use
    scripts/globals/pets.lua:298:5: (W314) value assigned to field GRAF is overwritten on line 488 before use
    scripts/globals/pets.lua:302:5: (W314) value assigned to field HERZOG is overwritten on line 815 before use
    scripts/globals/pets.lua:309:5: (W314) value assigned to field ZERO is overwritten on line 561 before use
    scripts/globals/pets.lua:313:5: (W314) value assigned to field MATAEMON is overwritten on line 1116 before use
    scripts/globals/pets.lua:383:5: (W314) value assigned to field ARCHER is overwritten on line 740 before use
    scripts/globals/pets.lua:384:5: (W314) value assigned to field SHOOTER is overwritten on line 1046 before use
    scripts/globals/pets.lua:432:5: (W314) value assigned to field ICE is overwritten on line 433 before use
    scripts/globals/pets.lua:896:5: (W314) value assigned to field SILVER is overwritten on line 897 before use
    scripts/globals/pets.lua:923:5: (W314) value assigned to field WINNER is overwritten on line 924 before use
    scripts/globals/pets.lua:1123:5: (W314) value assigned to field NORTHERN is overwritten on line 1124 before use
```

**Comment by mrhappyasthma** [comment]
Looks like this was introduced here: https://github.com/LandSandBoat/server/commit/8083de40b93044e029a4e3b181f6a05467c29588

**Comment by Snugglesffxi** [comment]
fixed by #390 ya?
